--- src/main.c	2019-09-26 21:37:14.000000000 +0000
+++ src/main.c	2020-03-20 20:43:20.876976754 +0000
@@ -425,9 +425,7 @@
 
 static void handle_sigterm(evutil_socket_t sock, short flags, void *arg)
 {
-	log_info("got SIGTERM, fast exit");
-	/* pidfile cleanup happens via atexit() */
-	exit(1);
+	log_info("Got SIGTERM, ignoring!");
 }
 
 static void handle_sigint(evutil_socket_t sock, short flags, void *arg)
